hw/top_earlgrey/dv/cov/pinmux_pad_attr.el

```diff
@@ -0,0 +1,2789 @@
+//==================================================
```

Comment:
This should have ideally been a pre-compile exclusion thing using the `-cm_constfile` approach, which lets VCS do constant propagation analysis and automatically exclude related signals (and metrics) in other hierarchies). Unfortunately, VCS does not support the addition of MDAs (most of our RTL ports are structs) in constfiles. 

The `-cm_hier` cfg file method is out of scope due to the large number of entities that need to be excluded. 

The `el` file approach turned out to be better because Verdi can exclude "related" signals automatically, so I just had to select these signals in one hierarchy and it found the related ones in other hierarchies.